src/shared/components/HeaderTextField/HeaderTextField.style.tsx

```diff
@@ -0,0 +1,68 @@
+import styled from '@emotion/styled'
+import { fluidRange } from 'polished'
+import { Text } from '@/shared/components'
+import { colors, sizes, typography, breakpoints } from '@/shared/theme'
+
+export const StyledInput = styled.input`
+  line-height: 1;
+  padding: ${sizes(1)} ${sizes(2)} ${sizes(2)};
+  ${fluidRange({ prop: 'fontSize', fromSize: '32px', toSize: '40px' })};
+  color: white;
+  background-color: ${colors.gray[800]};
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  overflow: hidden;
+  font-family: ${typography.fonts.headers};
+  font-weight: ${typography.weights.bold};
+  border: none;
+  min-width: 100px;
+  max-width: 100%;
+  height: ${sizes(13)};
+  @media screen and (min-width: ${breakpoints.small}) {
+    max-width: 600px;
+  }
+`
+
+export const WarningText = styled(Text)`
+  color: ${colors.warning};
+  background-color: ${colors.gray[800]};
+  max-width: 600px;
+  padding: ${sizes(2)} 0;
+  ${fluidRange({ prop: 'fontSize', fromSize: '10px', toSize: '16px' })};
+`
+
+export const StyledTooltip = styled.span`
```

Comment:
oh, also, tooltip isn't in scope for this task, there's another one specifically for creating tooltip element since we'll need that in a few places. So the tooltip will come from outside, from consumer. To not waste your work, you can move what you have here to `shared/components/WIP_Tooltip` so later we can use that as a starting point